cfg_VHcc.py

```diff
@@ -0,0 +1,55 @@
+# Local Variables:
+# python-indent-offset: 4
+# End:
+
+from VHcc.workflows.Zll_process import (
+    NanoProcessor as VH_Zll,
+)
+
+cfg = {
+    "userconfig": {'version':'test_nolepsf'},
+    "dataset": {
+        "jsons": ["src/VHcc/metadata/mcsamples_2017_vjets_Zll_used_nonCorruptedOnly.json",
+                  "src/VHcc/metadata/mcsamples_2017_vjets_ext_Zll_used_nonCorruptedOnly.json",
+                  "src/VHcc/metadata/mcsamples_2017_other_Zll_used_nonCorruptedOnly.json",
+                  "src/VHcc/metadata/mcsamples_2017_higgs_Zll_used.json",
+                  "src/VHcc/metadata/datasamples_2017_Zll_used.json"],
+        "campaign": "UL17",
```

Comment:
If I understand correctly, samples in metadata are Pre-UL 94X currently, but we will switch in the future. I don't expect this to be a big problem as of now, but we should still keep this in mind for relevant corrections, SFs etc.